python_lib/setup.py

```diff
@@ -0,0 +1,10 @@
+from setuptools import setup
+
+def readme():
+    with open('README.md') as f:
+        README = f.read()
+    return README
+
+setup(
+    
```

Comment:
Question: Is this suppose to be empty?

Reply:
I have @agentmilindu regarding the author details. So need to fill this up 😁

Reply:
ah good point